hedera-dependency-versions/build.gradle.kts

```diff
@@ -18,7 +18,7 @@ plugins {
     id("com.hedera.hashgraph.versions")
 }
 
-val besuNativeVersion = "0.6.1"
+val besuNativeVersion = "0.8.2"
```

Comment:
Have we confirmed that `0.8.2` of the native libraries is compatible with the besu `23.7.2` release? The besu `23.7.2` release currently depends on `0.8.1` of the native libraries as linked below.

https://github.com/hyperledger/besu/blob/7f959f2831014a3985e65ed94859bc5dd10cb3b3/gradle/versions.gradle#L163-L170

Reply:
I checked with Danno there are very few changes outside the additional method I added for this PR. Any none should effect compatibility.